emhttp/plugins/dynamix.vm.manager/novnc/app/styles/base.css

```
@@ -546,356 +555,352 @@ html {
+    background-color: var(--novnc-darkgrey);
+    border: none;
+    padding: 10px;
 }
 
 /* Shutdown/Reboot */
 :root:not(.noVNC_connected) #noVNC_power_button {
-  display: none;
+    display: none;
 }
 #noVNC_power {
 }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)